### DISCUSSION TITLE
Scratch Addons is Down?

### DISCUSSION THREAD

**Discussion by SunSpark647FromScratch** [ScratchAddons/ScratchAddons#2495] [Help] [answered]
On 5-14-21, Scratch Addons pop-up, settings, and addons added to Scratch are not working. Could anyone tell me why? I think it's down. I have version 1.14.2.

**Comment by retronbv** [comment]
There is no possible way for it to be down since installing the extension installs all of the code. This means it cannot be down since it is all on your local machine

**Reply by Explosion-Scratch** [reply]
lmao

**Comment by Explosion-Scratch** [comment]
Although as @retronbv it is indeed all local it's possible that there was an error in the code (go to `chrome://extensions` and check) or that you don't have the extension on. (also go to `chrome://extensions` to check), or that you have another app or extension interfering with scratchaddons.

**Reply by SunSpark647FromScratch** [reply]
well it's just showing this and nothing else:
![image](https://user-images.githubusercontent.com/82188343/118321904-e9427080-b4c3-11eb-9a2c-721c4bdee9a6.png)

**Reply by SunSpark647FromScratch** [reply]
you think it will help?

**Reply by Explosion-Scratch** [reply]
That is weird, I'd reload the extension and turn off other extensions

**Reply by SunSpark647FromScratch** [reply]
ok

**Reply by SunSpark647FromScratch** [reply]
nothing happened it's still showing that screen

**Reply by TheColaber** [reply]
@Explosion-Scratch you know nothing... 

@SunSpark647FromScratch go to `chrome://extensions` and turn on developer mode
![image](https://user-images.githubusercontent.com/72760579/118324778-7be90c80-b4d0-11eb-855d-877b34dac54a.png) Then look for scratch addons and click the errors button
![image](https://user-images.githubusercontent.com/72760579/118324812-886d6500-b4d0-11eb-8d74-cfac17a48c73.png)
Take a screenshot of that and send it here

**Reply by SunSpark647FromScratch** [reply]
hmmm....... there's no "Errors" button. All I see is this:
![image](https://user-images.githubusercontent.com/82188343/118359516-6c59da00-b549-11eb-98b7-2beef2b2c2b5.png)

**Reply by FunctionalMetatable** [reply]
> hmmm....... there's no "Errors" button. All I see is this:
> ![image](https://user-images.githubusercontent.com/82188343/118359516-6c59da00-b549-11eb-98b7-2beef2b2c2b5.png)

did you turn **on** developer mode

**Reply by SunSpark647FromScratch** [reply]
![image](https://user-images.githubusercontent.com/82188343/118372258-63cbc880-b576-11eb-8c9e-8ed72051d94c.png)

**Comment by SunSpark647FromScratch** [comment, marked as answer]
Okay, it's now fixed for me, no more worries!

**Reply by TheColaber** [reply]
Could you tell us what the problem was and how you fixed it?

**Reply by nexensys** [reply]
I think I know what it was. I have noticed the same issue some times myself when I load a dev version. The extension fails to load the addons sometimes and it shows up like that. I usually end up having to refresh the extension to get it to work again.
